src/kite.js

```diff
@@ -637,6 +637,12 @@ const Kite = {
         default:
           if (status) {
             switch (status.status) {
+              case "noindex":
+                this.statusBarItem.color = undefined;
+                this.statusBarItem.text = "𝕜𝕚𝕥𝕖: (unindexed)";
```

Comment:
could we make this `𝕜𝕚𝕥𝕖: ready (unindexed)`?

Reply:
Happy to make the change, but just for context, I opted to omit "ready" to keep this consistent with the preexisting status message, which just displays "𝕜𝕚𝕥𝕖" when ready.

Reply:
My concern is that users might mistake just `(unindexed)` for "something is broken" if we don't explicitly also say ready, but @its-dhung can chime in when he gets back.

We can merge the latest regardless, and change it later if necessary.